package.json

```diff
@@ -289,6 +290,7 @@
     "react-beautiful-dnd": "11.0.5",
     "react-dom": "16.14.0",
     "react-redux": "7.2.2",
+    "recharts": "2.0.9",
```

Comment:
I'm ok with this package in general since it's not adding any deps to an existing WebUI that would have performance concerns. But, before you merge, please could you request a basic security review on the security repo? It's a team policy, just in case there're any red flags. If you don't have access to this repo just let me know and I'll get it started.

Reply:
https://github.com/brave/security/issues/408